bindings/apns/apns.go

```diff
@@ -0,0 +1,159 @@
+package apns
```

Comment:
Can you please add license as comment in each code ?

Reply:
On the header, do I apply the standard with the Microsoft copyright (from the Kafka binding):

```
// ------------------------------------------------------------
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
// ------------------------------------------------------------
```

or do I leave off the Microsoft Copyright and just say "Licensed under the MIT License"?

Reply:
The headers may eventually blanket change under some governance changes that the project is undergoing, but for now please keep the MS header.